packages/kbn-monaco/src/console/lexer_rules/console_output.ts

```diff
@@ -15,4 +19,24 @@ export const consoleOutputLanguageConfiguration: monaco.languages.LanguageConfig
 
 export const consoleOutputLexerRules: monaco.languages.IMonarchLanguage = {
   ...consoleSharedLexerRules,
+  tokenizer: {
+    ...consoleSharedLexerRules.tokenizer,
+    comments: [
+      // Line comment indicated by #
+      // Everything after the # character is matched, stopping right before the status code and status text at the end if they are present
+      matchTokensWithEOL('comment', /# .+?(?=\s+\d{3}(?: \w+)*$)/, 'root', 'status'),
+      ...consoleSharedLexerRules.tokenizer.comments,
+    ],
+    status: [
+      // Following HTTP response status codes conventions
+      // Informational responses (status codes 100 – 199)
+      matchTokensWithEOL('status.info', /\b1\d{2}(?: \w+)*$/, 'root'),
+      // Successful responses (status codes 200 – 299)
+      matchTokensWithEOL('status.success', /\b2\d{2}(?: \w+)*$/, 'root'),
+      // Redirection messages (status codes 300 – 399)
+      matchTokensWithEOL('status.redirect', /\b3\d{2}(?: \w+)*$/, 'root'),
+      // Client and server error responses (status codes 400 – 599)
+      matchTokensWithEOL('status.error', /\b[4-5]\d{2}(?: \w+)*$/, 'root'),
```

Comment:
Grouped the status codes as per the [HTTP status code groups](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status). As far as I know, Es currently only uses 2**, 4**, and 5** status codes (200 OK, 201 Created, 202 Accepted, 204 No Content, 400 Bad Request, 401 Unauthorized, 403 Forbidden, 404 Not Found, 409 Conflict, 429 Too Many Requests, 500 Internal Server Error, 503 Service Unavailable). Practically only the `status.success` and `status.error` tokens will be used, but added the rest in case these status codes are added in Es.